hathor/p2p/sync_v2/agent.py

```diff
@@ -813,21 +833,23 @@ def handle_get_best_block(self, payload: str) -> None:
         """
         best_block = self.tx_storage.get_best_block()
         meta = best_block.get_metadata()
+        assert meta.validation.is_fully_connected()
+        # assert not meta.voided_by
```

Comment:
Remove or uncomment this?

Reply:
Same here. I'll leave a comment. I think this assert should be there but the current implementation does not work with it.

Reply:
I agree that we can remove it for now. Done! f880a62fac1873f7d8abb384f6209d1b919e6e46